docs/setup/rke2-cluster.md

```diff
@@ -0,0 +1,67 @@
+# Intro 
+
+## RKE2 Workload Cluster Deployment on Metal<sup>3</sup>
+
+This is a step by step guide on deploying RKE2 workload clusters on the SUSE Metal<sup>3</sup> Demo environment
+
+### Pre-requisites
+- A fully functioning Metal<sup>3</sup> deployment
+- At least 2 virtual bare metal hosts
+
+## Deploying the Workload Cluster
+
+1. Get the mac address of whichever VBMH will act as the control plane
+- Sample command: ```virsh dumpxml node-1 | grep mac```
```

Comment:
We can extract the MAC address via `$ virsh dumpxml node-1 | grep "mac address" | cut -d"'" -f 2`, store it into a variable and inline the file creation similarly to what we do in `vbmh-setup.md`.